build.sbt

```diff
@@ -28,7 +28,7 @@ name := "chisel-module-template"
 
 version := "3.2-SNAPSHOT"
 
-scalaVersion := "2.11.12"
+scalaVersion := "2.12.6"
```

Comment:
Isn't there supposed to be some kind of `-Xsource` flag that goes with this?

Reply:
There is a -Xsource:2.11 at line 8